source/2019-12-03-countdown-to-the-new-year-ember-modifier.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Countdown to The New Year- Ember Modifier
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+alias: "blog/2019/12/03/countdown-to-the-new-year-2019-ember-modifier"
+responsive: true
+---
+
+## Day 3
+
+This is the third in our DecEmber series– _"Countdown to The New Year: 31 Days of Ember Addons"_. We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!
+
+Today we are going to look at a rising star in our addon ecosystem- [ember-modifier](https://emberobserver.com/addons/ember-modifier). 
+
+READMORE
+
+### What It Does
+
+The `ember-modifier` addon provides an API for authoring element modifiers in Ember. It mirrors Ember's [helper API](https://api.emberjs.com/ember/3.14/classes/Ember.Templates.helpers), and has forms for writing both simple functional modifiers and more complicated class modifiers.
+
+A modifier can add customized, automatic behavior to a DOM element, such as tracking scroll position or adding new event listeners. After you write a modifier, you can use it in a template like this:
+
+```
```

Comment:
The markdown lint currently errors in regards to this code block and I think this would fix it:

```suggestion
```handlebars
```